### PR TITLE
Add instructions for using Typekit

### PR DESCRIPTION
**Pull request by misscs**
Typekit now supports CSS Loading 🙌 

- Add example of how to use Typekit

Related to https://github.com/ampproject/amphtml/issues/2820

### CODE REVIEW

File: spec/amp-html-format.md [hunk 1]
```diff
@@ -331,6 +331,7 @@ Font providers can be whitelisted if they support CSS-only integrations and serv
 - Fonts.com: `https://fast.fonts.net`
 - Google Fonts: `https://fonts.googleapis.com`
 - Font Awesome: `https://maxcdn.bootstrapcdn.com`
+- [Typekit](https://blog.typekit.com/2017/08/30/css-font-loading/): `https://use.typekit.net/kitId.css` (replace `kitId` accordingly)
```
Comment (ithinkihaveacat) [inline spec/amp-html-format.md]: This is probably a better link: https://helpx.adobe.com/typekit/using/google-amp.html

**Comment by aghassemi** [comment]
@misscs Thanks. Please take a moment to sign the CLA before we can merge. Thank you.

**Comment by misscs** [comment]
Hi Ali, most welcome. I’m having an issue signing the CLA because I don’t use my @gmail account on github. Is there a way I can use the secondary email on my google account? that is linked to my github.

File: spec/amp-html-format.md (hunk 1 above)
Comment (aghassemi) [reply spec/amp-html-format.md]: agree, updated to https://helpx.adobe.com/typekit/using/google-amp.html

**Comment by aghassemi** [comment]
Thanks @misscs, CLA is good. Merging.

**Comment by misscs** [comment]
Thanks, @aghassemi!

**Pull request merged**